app/modules/Observer.rb

```diff
@@ -0,0 +1,3 @@
+module Observer
+  def new_notification(object); end
```

Comment:
why is this method blank? Could we add a comment like:
```suggestion
  # TODO: Implement bla bla bla
  def new_notification(object); end
```